blaze/compute/tests/test_pandas_compute.py

```diff
@@ -45,6 +45,20 @@
                   columns=['name', 'sex', 'amount', 'id'])
 
 
+def test_series_broadcast():
+    s = Series([1, 2, 3], name='a')
+    t = symbol('t', 'var * {a: int64}')
+    bcast = broadcast_collect(expr=(2 * t.a - t.a**2))
+    result = compute(bcast, s)
+    assert_series_equal(2 * s - s**2, result)
+
+
+def test_frame_broadcast():
+    bcast = broadcast_collect(expr=t.amount * t.id)
+    result = compute(bcast, df)
+    assert_series_equal(df.amount * df.id, result)
```

Comment:
can we swap the assertion to make the lhs the result and the rhs the expected? I know that this is equivalent but that pattern is more common